[blog] Danail Hadjiatanasov joins Material-UI (#23223)
mui-org/material-ui: docs/pages/blog/danail-hadjiatanasov-joining.md

---
title: Danail Hadjiatanasov joins Material-UI

> description: We are excited to share that Danail Hadjiatanasov has joined Material-UI. It's his first full-time week and is now part of the enterprise team.

Reviewer: ```suggestion
description: We are excited to share that Danail Hadjiatanasov has joined Material-UI as part of the enterprise team. This was his first full-time week.
```

date: 2020-10-23T00:00:00.000Z
authors: ['oliviertassinari']
card: true
---

> We are excited to share that [Danail Hadjiatanasov](https://twitter.com/danail_h) has joined Material-UI. It's his first full-time week and is now part of the enterprise team.

Reviewer: ```suggestion
We are excited to share that [Danail Hadjiatanasov](https://twitter.com/danail_h) has joined Material-UI as part of the enterprise team. This was his first full-time week.
```

> For the past 3 years, Danail has been working in a company called [Backbase](https://www.backbase.com/). He was responsible for building a UI library based on Angular that was specifically designed for banks. The UI library and its corresponding design system were sold under a commercial license and today some of the biggest names in the banking industry have web applications powered by those UI components (RBC, Central 1, Deutsche Bank, BNP Paribas, and others). Although he was working with Angular his true passion is React hacking side projects with it. He's extremely happy to be part of Material-UI’s team and help expand the product even more.

Reviewer: ```suggestion
For the past 3 years Danail has been working in a company called [Backbase](https://www.backbase.com/) where he was responsible for building a UI library based on Angular that was specifically designed for banks. The UI library and its corresponding design system were sold under a commercial license, and today some of the biggest names in the banking industry have web applications powered by those UI components (RBC, Central 1, Deutsche Bank, BNP Paribas, and others). 

Although he was working with Angular, his true passion is React, having hacked side projects with it. now he will be able to combine these two talents, and help take Material-UI X to the next level!
```

> Danail is off to a running start, he has landed his first feature in the data grid: the column [reordering](https://material-ui.com/components/data-grid/columns/#column-reorder):

Reviewer: ```suggestion
Danail has already landed his first feature in the data grid: [column reordering](https://material-ui.com/components/data-grid/columns/#column-reorder):
```

---
title: Danail Hadjiatanasov joins Material-UI
description: We are excited to share that Danail Hadjiatanasov has joined Material-UI as part of the enterprise team. This was his first full-time week.
date: 2020-10-23T00:00:00.000Z
authors: ['oliviertassinari']
card: true
---

We are excited to share that [Danail Hadjiatanasov](https://twitter.com/danail_h) has joined Material-UI as part of the enterprise team. This was his first full-time week.

For the past 3 years Danail has been working in a company called [Backbase](https://www.backbase.com/) where he was responsible for building a UI library based on Angular that was specifically designed for banks. The UI library and its corresponding design system were sold under a commercial license, and today some of the biggest names in the banking industry have web applications powered by those UI components (RBC, Central 1, Deutsche Bank, BNP Paribas, and others). 

> Although he was working with Angular, his true passion is React, having hacked side projects with it. now he will be able to combine these two talents, and help take Material-UI X to the next level!

Reviewer: ```suggestion
Although he was working with Angular, his true passion is React, having hacked side projects with it. Now he will be able to combine these two talents, and help take Material-UI X to the next level!
```
